### PR TITLE
Combined landing pages

### PR DESCRIPTION
**Pull request by KimberlyMunoz**
I also adjusted styles to reduce duplicate markup and make missing adjustments to the molecules. If there's other adjustments or things that need to be brought in line with each other, let me know.

## Changes

- Brings 50-50 molecule in line with the styles in the half link blob. Previously, this was using the content-l styles.
- Adds title variable to the sidebar molecules
- Combines the two landing pages into one landing page

## Preview

image
image
image

## Review
- @sebworks 
- @jimmynotjim

### CODE REVIEW

File: cfgov/jinja2/v1/landing-page/index.html [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+{% import 'molecules/hero.html' as hero %}
+{% import 'molecules/half-width-link-blob.html' as link_blob %}
+{% import 'molecules/image-text-25-75.html' as image_text_25_75 %}
+{% import 'molecules/call-to-action.html' as call_to_action %}
+{% import 'organisms/well.html' as well %}
+{% import 'organisms/email-signup.html' as email_signup %}
+{% import 'molecules/related-posts.html' as related_posts %}
+{% import 'molecules/text-introduction.html' as text_introduction %}
+{% import 'molecules/image-text-50-50.html' as image_text_50_50 %}
+{% import 'molecules/related-posts.html' as related_posts %}
```
Comment (jimmynotjim) [inline cfgov/jinja2/v1/landing-page/index.html]: Can we group the organisms and molecules together like we do w/ the CSS to make it easier to find?

File: cfgov/jinja2/v1/landing-page/index.html [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+    }) }}
+{% endblock %}
+
+{# TODO: Remove lipsumText when and if it is no longer needed. #}
+{% set lipsumText = lipsum(1, false, 10, 25) | safe %}
+
+{% block content_main %}
+     <div class="block
+                block__flush-top">
+        {# This molecule should always be first #}
```
Comment (jimmynotjim) [inline cfgov/jinja2/v1/landing-page/index.html]: This block looks like it's short of 1 space in the indentation

File: cfgov/jinja2/v1/landing-page/index.html [hunk 3]
```diff
@@ -0,0 +1,161 @@
...
+    </div>
+    <div class="block">
+        <h2>Headline for 2/7 organism</h2>
+        {{ image_text_25_75.render({
+            'image_path': 'http://placehold.it/300x300',
+            'image_alt': '',
+            'heading' : 'Placeholder',
+            'body': lipsumText,
+            'link_url': '#',
+            'link_text' : 'Placeholder link'
```
Comment (jimmynotjim) [inline cfgov/jinja2/v1/landing-page/index.html]: There's an extra space before the `:` here and on the next three 25/75 blocks

File: cfgov/jinja2/v1/landing-page/index.html [hunk 4]
```diff
@@ -0,0 +1,161 @@
...
+        }) }}
+    </div>
+
+    <div class="block">
+            {{ well.render( 'landing-page/_template-prototype-well-content.html' ) }}
```
Comment (jimmynotjim) [inline cfgov/jinja2/v1/landing-page/index.html]: Remove extra indentation here

File: cfgov/jinja2/v1/landing-page/index.html [hunk 5]
```diff
@@ -0,0 +1,161 @@
...
+
+{# TODO: Remove lipsumText when and if it is no longer needed. #}
+{% set lipsumText = lipsum(1, false, 10, 25) | safe %}
+
+{% block content_main %}
+     <div class="block
+                block__flush-top">
+        {# This molecule should always be first #}
+        {{ text_introduction.render({
+            'heading': 'Text Introduction',
```
Comment (jimmynotjim) [inline cfgov/jinja2/v1/landing-page/index.html]: Should we align values throughout? We have a mix of aligned and unaligned

File: cfgov/unprocessed/css/organisms/image-text-50-50-group.less [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+      codenotes:
+        - |
+          Structural cheat sheet:
+          -----------------------
+          .o-image-text-50-50
+  tags:
+    - cfgov-molecules
+*/
+.o-image-text-50-50-group {
+  .respond-to-min(@bp-sm-min, {
```
Comment (jimmynotjim) [inline cfgov/unprocessed/css/organisms/image-text-50-50-group.less]: CSS indentation should be 4 spaces

**Comment by jimmynotjim** [comment]
Is the 50/50 supposed to be pulling left and right by 15px at large devices? cc @duelj @ajbush 

screen shot 2015-11-18 at 2 32 47 pm

**Comment by KimberlyMunoz** [comment]
I actually I'm not 100% sure about that. I copied over the styles from half link blob, but now it seems out-of-alignment with the other molecules.

File: cfgov/jinja2/v1/landing-page/index.html (hunk 1 above)
Comment (KimberlyMunoz) [inline cfgov/jinja2/v1/landing-page/index.html]: That's a good idea, I found a duplicate import in there. :dancers:

Comment (KimberlyMunoz) [inline cfgov/jinja2/v1/landing-page/index.html]: Fixed in 3919ee5

File: cfgov/jinja2/v1/landing-page/index.html (hunk 2 above)
Comment (KimberlyMunoz) [inline cfgov/jinja2/v1/landing-page/index.html]: Fixed in d2bb0c4

File: cfgov/jinja2/v1/landing-page/index.html (hunk 4 above)
Comment (KimberlyMunoz) [inline cfgov/jinja2/v1/landing-page/index.html]: Fixed in d2bb0c4

File: cfgov/jinja2/v1/landing-page/index.html (hunk 5 above)
Comment (KimberlyMunoz) [inline cfgov/jinja2/v1/landing-page/index.html]: Fixed in 3919ee5. Let me know if there's any that are over-aligned!

File: cfgov/jinja2/v1/landing-page/index.html (hunk 3 above)
Comment (KimberlyMunoz) [inline cfgov/jinja2/v1/landing-page/index.html]: Fixed in  3919ee5

File: cfgov/unprocessed/css/organisms/image-text-50-50-group.less (hunk 6 above)
Comment (KimberlyMunoz) [inline cfgov/unprocessed/css/organisms/image-text-50-50-group.less]: Copied this from another file so I fixed it here and there in d2bb0c4

**Comment by jimmynotjim** [comment]
:+1:

**Pull request merged**